project/.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,52 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
We don't really need files from your IDE.

Reply:
deleted

Reply:
> `project` - isn't the self-descriptive name for your application.

renamed